plugins/adr-backend/src/search/DefaultAdrCollatorFactory.ts

```diff
@@ -117,7 +123,8 @@ export class DefaultAdrCollatorFactory implements DocumentCollatorFactory {
     this.parser = options.parser ?? createMadrParser();
     this.reader = options.reader;
     this.scmIntegrations = ScmIntegrations.fromConfig(options.config);
-    this.tokenManager = options.tokenManager;
+
+    this.auth = createLegacyAuthAdapters(options).auth;
```

Comment:
I assuming once we drop support for the old backend system along with fully deprecating the `TokenManagerService` and `IdentityService` we will need to make a follow up breaking change to remove these usages of `createLegacyAuthAdapters` to directly use the new auth services instead?

Reply:
Not strictly so; `createLegacyAuthAdapters` is smart enough to just plain pass through the auth and httpAuth services when they are given. But yeah in practice, once you remove the old deprecated types on the options object, you'll probably just choose to remove the adapter call at the same time.